doc improvement: style fixes in Other samples (#6797)
nrfconnect/sdk-nrf: samples/peripheral/lpuart/README.rst

Additionally, it requires a logic analyzer.

Overview
********

The sample implements a simple loopback using a single UART instance.

> It has console and logging disabled by default, to demonstrate low power consumption while having UART active.

Reviewer: ```suggestion
By default, the console and logging are disabled to demonstrate low power consumption when UART is active.
```
